### ISSUE TITLE
Add OpenSUSE Tumbleweed install_dependencies.sh script or placeholder

### ISSUE THREAD

**Issue by leblowl** [openframeworks/openFrameworks#6769]
Here are some of the dependencies that I needed to install in order to get the project to compile:

```
zypper in \
alsa-devel \
freeglut-devel \
freeimage-devel \
gcc-c++ \
glew-devel \
gstreamer-devel \
gstreamer-plugins-base-devel \
libglfw-devel \
libpulse-devel \
libsndfile-devel \
libudev-devel \
libXrandr-devel \
openal-soft-devel \
pugixml-devel \
rtaudio-devel \
uriparser-devel
```

This is probably not a complete list, since I most likely had some of the dependencies installed already, but it's a start. Would a WIP install script (or README) be an acceptable addition to the project, or would you prefer a complete working install script? I can launch a VM and get the full list pretty easily.

**Comment by arturoc** [comment]
Yes having a full list would be the most useful. A PR with a working script similar to the ones for other platforms would be the preferred format but if adding a full list to this issue if you can't send a PR with the full script might help others using OpenSUSE finding out how to install the dependencies
